Meme/Meme/ViewController.swift

```diff
@@ -10,10 +10,37 @@ import UIKit
 
 class ViewController: UIViewController {
 
+    var networkClient: NetworkClient
```

Comment:
Let's use a view model. And refer to the networkClient via its protocol.

Reply:
Hmm, the network client doesn't currently define a protocol. I've taken a stab at this, but let me know if you think otherwise.